rskj-core/src/main/java/co/rsk/peg/ErpFederation.java

```diff
@@ -131,7 +62,6 @@ public boolean equals(Object other) {
         }
 
         ErpFederation otherErpFederation = (ErpFederation) other;
-
         return this.getNumberOfSignaturesRequired() == otherErpFederation.getNumberOfSignaturesRequired() &&
```

Comment:
Didn't we agree on checking the address?

Reply:
ok!! wasn't sure